dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/rates/DeleteRatesSpec.scala

```diff
@@ -0,0 +1,59 @@
+package com.wavesplatform.it.matcher.api.http.rates
+
+import com.softwaremill.sttp.StatusCodes
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.api.http.entities.HttpRates
+import com.wavesplatform.dex.domain.asset.Asset.Waves
+import com.wavesplatform.dex.it.api.RawHttpChecks
+import com.wavesplatform.dex.it.docker.apiKey
+import com.wavesplatform.it.MatcherSuiteBase
+
+class DeleteRatesSpec extends MatcherSuiteBase with RawHttpChecks {
+
+  val defaultRates: HttpRates = Map(Waves -> 1d)
+
+  override protected def dexInitialSuiteConfig: Config = ConfigFactory.parseString(
+    s"""waves.dex {
+       |  price-assets = [ "$BtcId", "$UsdId", "WAVES" ]
+       |}""".stripMargin
+  )
+
+  override protected def beforeAll(): Unit = {
+    wavesNode1.start()
+    broadcastAndAwait(IssueBtcTx, IssueUsdTx)
```

Comment:
It looks like, BTC is not used. Consider remove it and from config too.

Reply:
fixed